src/pytest_springfield/dottedname/foo/bar/bop.py

```diff
@@ -0,0 +1,14 @@
+from springfield import fields, Entity
+
+
+class Property(Entity):
+    name = fields.StringField()
+    nested = fields.EntityField(
+        'pytest_springfield.dottedname.foo.bar.bop.PropertyList'
```

Comment:
I can see this as being abused. Generally we like symbols for references instead of strings. However I can see the need for it and don't really have any better alternatives.